internal/pyxis/pyxis.go

```diff
@@ -431,6 +435,14 @@ func (p *pyxisClient) createTestResults(ctx context.Context, testResults *TestRe
 	}
 
 	if ok := checkStatus(resp.StatusCode); !ok {
+		// checking to see if the users is using an unsupported versions
+		// if so display preflights latest download url to the user
+		if resp.StatusCode == http.StatusBadRequest &&
+			(strings.Contains(string(body), "not recognized") || strings.Contains(string(body), "not supported")) {
+			logger.Error(errors.New("invalid preflight version, please please download the latest version and re-submit"),
```

Comment:
Are we super begging them to do it?

Reply:
Yes, we are being extra nice! I'll remove the dup! Good catch!